lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -229,13 +228,8 @@ export class UserAgentApplication {
    * @param {Array.<string>} scopes - Permissions you want included in the access token. Not all scopes are guaranteed to be included in the access token returned.
    * @param {string} extraQueryParameters - Key-value pairs to pass to the authentication server during the interactive authentication flow.
    */
-  // TODO: params to accept AuthRequest object instead
-  loginRedirect(scopes?: Array<string>, extraQueryParameters?: string): void {
-    /*
-    1. Create navigate url
-    2. saves value in cache
-    3. redirect user to AAD
-     */
+  loginRedirect(request: AuthenticationParameters): void {
```

Comment:
What happens if request === null?

Reply:
We will go with default scopes = clientId I believe. I can refer this and confirm my supposition.

Reply:
Confirmed that is the case.